phenol-cli/src/main/java/org/monarchinitiative/phenol/cli/demo/HpDemo.java

```diff
@@ -79,7 +80,7 @@ public void run() {
       System.out.println("\t" + sset);
     }
 
-    Map<TermId, HpoDisease> diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(annotPath,hpo);
+    Map<TermId, HpoDisease> diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(Paths.get(annotPath), hpo);
```

Comment:
I think `Path.of()` is the Java 11+ way

Reply:
Yeah, that's why this is super annoying and why I'd love to move phenol to Java 11+ soon.